spec/models/tenant_spec.rb

```diff
@@ -24,6 +24,10 @@
         expect(template.tenant_id).to eq tenant.id
       end
     end
+
+    after do
+      ActsAsTenant.current_tenant = nil
```

Comment:
@bzwei If using the with_tenant you don't need this
https://www.rubydoc.info/gems/acts_as_tenant/0.3.9/ActsAsTenant.with_tenant

Reply:
@mkanoor This is the part we mentioned during standup. This `after` will cleanup leftover tenants.

Reply:
Yes, we should use `with_tenant`. Fixed.